utils/argent-context.js

```diff
@@ -0,0 +1,143 @@
+/* global artifacts */
+
+const ethers = require("ethers");
+const BN = require("bn.js");
+
+const utils = require("./utilities.js");
+const RelayManager = require("./relay-manager");
+
+const WalletFactory = artifacts.require("WalletFactory");
+const BaseWallet = artifacts.require("BaseWallet");
+const ModuleRegistry = artifacts.require("ModuleRegistry");
+const TransferStorage = artifacts.require("TransferStorage");
+const GuardianStorage = artifacts.require("GuardianStorage");
+const ArgentModule = artifacts.require("ArgentModule");
+const DappRegistry = artifacts.require("DappRegistry");
+const ERC20 = artifacts.require("TestERC20");
+
+const { ETH_TOKEN } = utils;
+const ZERO_ADDRESS = ethers.constants.AddressZero;
+const SECURITY_PERIOD = 2;
+const SECURITY_WINDOW = 2;
+const RECOVERY_PERIOD = 4;
+const LOCK_PERIOD = 4;
+
+let tokensInitialized = false;
+
+class ArgentContext {
+  constructor(accounts) {
+    [
+      this.infrastructure,
+      this.owner,
+      this.guardian1,
+      , // eslint-disable-line
+      this.relayer, // 4
+      , // eslint-disable-line
+      this.tokensAddress, // 6
```

Comment:
this could maybe be renamed into something like `tokensHolder` to make the intend more clear

Reply:
done